content/altinn-studio/guides/options/_index.nb.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Kodelister (options)
+title: Kodelister (svaralternativer)
 linktitle: Kodelister
-description: Hvordan konfigurere options/kodelister for en app?
+description: Hvordan konfigurere svaralternativer/kodelister for en app?
 toc: true
 weight: 40
 ---
 
-Altinn tilbyr to ulike måter en app kan eksponere kodelister - statisk og dynamisk. Disse eksponeres primært fra options endepunktet i appen, og er tilgjengelig på `{org}/{app}/api/options/{optionsId}`.
-Checkbox, Dropdown og RadioButton komponenter vil automatisk kunne hente ut en slik liste om man kobler denne komponenten til en slik options-id. Men ikke alle dynamiske kodelister må gå via api'et - vi har også dynamiske kodelister som baserer seg på verdiene fra en repeterende struktur i datamodellen.
+Altinn tilbyr to ulike måter en app kan eksponere kodelister på: Statisk og dynamisk. Disse eksponeres primært fra endepunktet som er tilgjengelig på `{org}/{app}/api/options/{optionsId}`, hvor `optionsId` er ID-en til listen.
+Komponenter som avkrysningsbokser, radioknapper og nedtrekkslister vil automatisk kunne hente ut en slik liste om man kobler dem til en kodeliste-ID. Men ikke alle dynamiske kodelister må gå via API'et – vi har også dynamiske kodelister som baserer seg på verdiene fra en repeterende struktur i datamodellen.
```

Comment:
Det er flere komponenter som kan hente verdiene i kodelister når de blir koblet til en kodeliste-ID, blant annet avkrysningsbokser, radioknapper og nedtrekkslister. Men ikke alle dynamiske kodelister må gå via API-et. Vi har også dynamiske kodelister som er basert på verdiene fra en repeterende struktur i datamodellen.

Kommentar: Forkortelser i bestemt form skal ha bindestrek på norsk, så API-et, PC-en, ikke andre tegn. 

Den siste setningen med repeterende struktur i datamodellen, er heller ikke klar for en ny bruker som meg - kan du skrive om/forklare mer, eventuelt?